client/packages/android/capacitor.config.ts

```diff
@@ -6,14 +6,16 @@ import { CapacitorConfig } from '@capacitor/cli';
 const config: CapacitorConfig = {
   appId: 'org.openmsupply.client',
   appName: 'openmsupply-client',
-  // This is only needed for `npx cap copy` to work, it doesn't have to point to actual bundle
+  // This is only needed for `npx cap copy` to work, and it does have to point to actual bundle
   // bundle is server by remote server (local or discovered) or through webpack if debugging (see comment below)
-  webDir: 'notexistent',
+  webDir: '../host/dist/',
```

Comment:
was this a required change ? I remember doing this to avoid confusion between how capacitor usually works to how we use it (we use remote bundle vs local bundle for client to auto upgrade when server upgraded, and to avoid building another bundle for discovery).

I think technically we would still be using remote bundle even with this change, but if it's not required for this PR to work, can we keep at as `notexistent`, otherwise some can assume that capacitor app uses bundled app by capacitor rather then server.

Reply:
if the directory specified here doesn't exist, then capacitor doesn't build - at least, that's what happened for me, and is why I changed to the dist dir.
I updated the comment above the line to reflect this - and tested that the app still serves the remote app correctly